packages/rs-dpp/src/errors/consensus/basic/basic_error.rs

```diff
@@ -230,6 +230,9 @@ pub enum BasicError {
     #[error(transparent)]
     InvalidAssetLockProofTransactionHeightError(InvalidAssetLockProofTransactionHeightError),
 
+    #[error(transparent)]
+    InvalidAssetLockProofValueError(InvalidAssetLockProofValueError),
```

Comment:
This error is used in state validation function so it should be defined as state error.

Reply:
Updated